enhancements/console/workflow-guides.md

```diff
@@ -0,0 +1,175 @@
+      * Section - string - Defines area of interest, e.g. `operators`.
+      * Admin - bool - Defines if the Workflow Guide is cluster admin oriented (user has to have cluster admin permisions).
+      * Version - string - Defines the version of the `WorkflowGuide`
+      * **Extenstion:**: ResourceCheckName - string - Defines the name of the `ResourceCheck` CR.
+
+3. RedHat supported Workflow Guides will be places in new `openshift/workflow-guides` repository. Each root folder in the repository will be named after area of interest (e.g. top items from the left navigation menu -  Operators, Workloads, Serverless, ..., Administration).
+4. The `openshift/workflow-guides` repository will also contain a `manifests/` root folder that will contain all yaml for all the official Workflow Guides `ConsoleLinks` CRs. All the manifests will be copied into the `console-operator` image upon it's build. Alternativly there could be a small `workflow-guides-operator` that will be:
+   1. Contain all the `ConsoleLinks` CRs from the `manifests/` root folder.
+   2. Responsile for removing old `ConsoleLinks` CRs (based on the cluster version) after a cluster update, so the `ConsoleLink` CR points to a proper version of the `Workflow Guide` on GitHub.
+5. The `openshift/workflow-guides` repository will contain a script that would be responsible for updating all the `ConsoleLinks` CRs to point to the current version.
+6. All Workflow Guides will be listed available in a separate page that will be accessible from Help Menu.
+   ![help-menu](https://raw.githubusercontent.com/jhadvig/images/master/help-menu.png)
+7. For different areas of interest, the links for Workflow Guides will need to be handles separately:
+   * Operators - CVS of the operator that has a Workflow Guide published will contain `console.openshift.io/workflow-guide: <console-link-name>` annotation. Upon the operator installation an "Workflow Guide" link will appear next to the `Install` button.
+   ![operators](https://raw.githubusercontent.com/jhadvig/images/master/operators.png)
+   * ...TBD
+
+#### Resource Checker Extension
+
+In order for the user to see what resources need to be created in terms of a specific Workflow Guide, we will display list of those resources in a right sidebar similar to the proposed [Solution Explorer](https://marvelapp.com/236ge4ig/screen/69628466). Each of the resource items will be prefixed with an icon which will indicate if the resource exists or not.
+
+For the Resource Check we will need to define a new CRD that will contain list of resources. For each resource, following fields will need to be defined:
+   * Kind - string - Kind of the resource.
+   * Name - string - Name of the resource.
+   * Namespace - string - Namespace of the resource. (Optional) 
+   * CurrentNamespace - bool - Indicates whether the resource should be check in user's current namespace. If set will be honored instead of the `Namespace` field. (Optional) 
```

Comment:
Should we just assume the current namespace if no namespace is provided?

Reply:
right but there are also non-namespaced resources, which might need to be referenced. right ?